### PR TITLE
[WIP] add more configurations for perf benchmark job

### PR DESCRIPTION
**Pull request by carolynhu**
This PR is to solve: https://github.com/istio/istio/issues/17585
The fixed number of configurations align with this blog post: https://istio.io/blog/2019/performance-best-practices/

Currently, this is not ready for review.

### CODE REVIEW

**Comment by carolynhu** [comment]
@richardwxn yes, we should, I'm busying with ui stuff today, if you get a time, please fix this.

File: perf/benchmark/run_benchmark_job.sh [hunk 1]
```diff
@@ -78,26 +78,36 @@ popd
 setup_metrics
 echo "Start running perf benchmark test."
 # For adding or modifying configurations, refer to perf/benchmark/README.md
-# Configuration1
+# Configuration Set1: CPU and memory
```
Comment (richardwxn) [inline perf/benchmark/run_benchmark_job.sh]: * with mixer enabled

File: perf/benchmark/run_benchmark_job.sh [hunk 2]
```diff
@@ -78,26 +78,36 @@ popd
...
+DURATION=240
 METRIC="cpu"
 # shellcheck disable=SC2086
 pipenv run python3 runner.py ${CONN} ${QPS} ${DURATION} ${EXTRA_ARGS}
 collect_metrics true ${METRIC}
 METRIC="mem"
 collect_metrics true ${METRIC}
 
-# Configuration2
+# Configuration Set2: Latency Quantiles
```
Comment (richardwxn) [inline perf/benchmark/run_benchmark_job.sh]: same as above

File: perf/benchmark/run_benchmark_job.sh [hunk 3]
```diff
@@ -78,26 +78,36 @@ popd
...
+# shellcheck disable=SC2086
+METRIC="p90"
+collect_metrics true ${METRIC}
+# shellcheck disable=SC2086
+METRIC="p99"
+collect_metrics true ${METRIC}
+
+# Configuration Set3: CPU and memory with mixer disabled
+
+# Configuration Set4: Latency Quantiles with mixer disabled
```
Comment (richardwxn) [inline perf/benchmark/run_benchmark_job.sh]: Set5 and 6 should be CPU/memory and latency with mixer v2 enabled

**Comment by carolynhu** [comment]
in favor of: https://github.com/istio/tools/pull/414

**Pull request closed without merging**